fe/fe-core/src/main/java/com/starrocks/persist/EditLog.java

```diff
@@ -292,10 +292,13 @@ public void loadJournal(GlobalStateMgr globalStateMgr, JournalEntity journal)
                 }
                 case OperationType.OP_DROP_PARTITION: {
```

Comment:
Do not add code logic here, you should add all the logic into` LocalMetastore. replayDropPartition`